tutorials/intro/plot_object_annotations.py

```diff
@@ -1,245 +1,234 @@
+#
+# Many EEG and MEG systems incorporate one or more recording channels that do
+# not receive signals from a sensor; instead, those channels record voltages
+# (usually short, rectangular DC pulses of fixed magnitudes sent from the
+# experiment-controlling computer) that are time-locked to experimental events,
+# such as the onset of a stimulus or a button-press response by the subject. In
+# other cases, these pulses may not be strictly time-locked to an experimental
+# event, but instead may occur in between trials to indicate the type of
+# stimulus (or experimental condition) that is about to occur on the upcoming
+# trial. In MNE-Python, those channels are referred to as :term:`"STIM"
+# channels <stim channel>`.
+#
+# The DC pulses may be all on one STIM channel (in which case different
+# experimental events or trial types are encoded as different voltage
+# magnitudes), or they may be spread across several channels, in which case the
+# channel(s) on which the pulse(s) occur can be used to encode different events
+# or conditions. Even on systems with multiple STIM channels, there is often
+# one channel that records a weighted sum of the other STIM channels, in such a
+# way that voltage levels on that channel can be unambiguously decoded as
+# particular event types. On older Neuromag systems (such as that used to
+# record the sample data) this was typically the ``STI 014`` channel; on newer
+# systems it is more commonly ``STI101``. You can see the STIM channels in the
+# raw data file here:
+
+raw.copy().pick_types(meg=False, stim=True).plot(start=3, duration=6)
 
 ###############################################################################
-
-for idx, my_annot in enumerate(annot):  # iterate on the Annotations object
-    print('annot #{0}: onset={1}'.format(idx, my_annot['onset']))
-    print('annot #{0}: duration={1}'.format(idx, my_annot['duration']))
-    print('annot #{0}: description={1}'.format(idx, my_annot['description']))
+# Above, you can see that ``STI 014`` (the "sum channel") contains pulses of
+# different magnitudes whereas pulses on other channels have consistent
+# magnitudes. You can also see that every time there is a pulse on ``STI 001``
+# through ``STI 006`` there is a corresponding pulse on ``STI 014``.
+#
+#
+# Converting a STIM channel signal to an Events array
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# If your data has events recorded on a STIM channel, you can convert them into
+# an events array using :func:`mne.find_events`. The sample number of the onset
+# (or offset) of each pulse is recorded as the event time, the pulse magnitudes
+# are converted into integers, and these pairs of sample numbers plus integer
+# codes are stored in :class:`NumPy arrays <numpy.ndarray>` (usually called
+# "the events array" or just "the events"). In its simplest form, the function
+# requires only the :class:`~mne.io.Raw` object, and the name of the channel(s)
+# from which to read events:
```

Comment:
not sure if this goes into this tutorial but something that I missed early on when beginning in MNE was that I couldn't find the term "trigger" anywhere because that's what is used in many labs and also in FieldTrip. Perhaps incorporating that somewhere might be nice for people coming from other software

Reply:
it's in the big overview tutorial we just merged in last week. second sentence of this paragraph: https://mne-tools.github.io/dev/auto_tutorials/intro/plot_introduction.html#detecting-experimental-events